ecms_base/modules/custom/ecms_health_file_migration/config/install/migrate_plus.migration.health_file_json_to_resource_link.yml

```diff
@@ -0,0 +1,92 @@
+langcode: en
+status: true
+dependencies: {  }
+id: health_file_json_to_resource_link
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: null
+migration_group: files_json_to_resource_link
+label: 'Import links from JSON source to Resource Link type Media entities'
+source:
+  plugin: url
+  data_fetcher_plugin: http
+  urls:
+    - 'https://ecms-distribution.lndo.site/sites/default/files/resource_link/resource_link_sample.json'
```

Comment:
Is this supposed to be the lando url?

Reply:
Ah good catch - we are waiting on final URL from them. This will need to be updated before it is run on prod

Reply:
Got it